wheatley/row_generation/place_notation_generator.py

```diff
@@ -18,8 +18,8 @@ class PlaceNotationGenerator(RowGenerator):
     DefaultSingle: ClassVar[CallDef] = CallDef({0: '1234'})
 
     def __init__(self, stage: int, method: str, bob: CallDef = None, single: CallDef = None,
-                 start_index: int = 0) -> None:
-        super().__init__(stage)
+                 start_index: int = 0, start_row: str = None) -> None:
```

Comment:
Shouldn't this be `start_row: Option[str] = None`, since `str` can't be `None`.  I'm honestly surprised that the type checker isn't catching this, but it happens many times across the code (I think grepping for `str = None` should find them all) and it would probably be good to be explicit about it.

Reply:
I recall something about the typechecker allowing this in `__init__` (see the `bob: CallDef = None` ones )